src/services/legal-services.ts

```diff
@@ -357,8 +357,12 @@ export default class LegalServices {
     const url = `businesses/${businessId}/digitalCredentials`
     return axios.get(url)
       .catch(error => {
-        // eslint-disable-next-line no-console
-        console.log(error.message)
+        // display error in console if not HTTP 401
+        if (error?.response?.status !== StatusCodes.UNAUTHORIZED) {
```

Comment:
HTTP 401 is the response if the user if not allows to fetch digital credentials. No need to log an error for that.